configure-places-in-the-sdk/places-extension/using-places-without-active-region-monitoring.md

```diff
@@ -0,0 +1,415 @@
+    }
+}
+```
+{% endtab %}
+
+{% endtabs %}
+
+## Trigger Entry Events When the User is in a POI
+
+The SDK will give you back a list of nearby POIs, including whether or not the user is currently within each of the POIs. If the user is in a POI, you can have the SDK trigger an entry event for that region.
+
+{% hint style="important" %}
+In order to prevent your app from triggering multiple entry events, you will need to keep a list of regions that you already know the user to be within. When processing the response of nearby POIs from the SDK, you will only trigger an entry event if you haven't already triggered one for the region in question.
+
+In the example below, this is accomplished using `NSUserDefaults` (iOS) and `SharedPreferences` (Android).
+{% endhint %}
+
+{% tabs %}
+
+{% tab title="Android (Java)" %}
+
+This code sample shows handling the result provided in the callback of `getNearbyPointsOfInterest`, a `List<PlacesPOI>`:
+
+```java
+void handleUpdatedPOIs(final List<PlacesPOI> nearbyPois) {
+    // get the list of regions we know the user is already within from SharedPreferences
+    SharedPreferences preferences = getApplicationContext().getSharedPreferences("places", 0);
+    Set<String> regionsUserIsAlreadyIn = preferences.getStringSet("regionsUserIsAlreadyIn", new HashSet<String>());
+
+    // loop through new placesPOIS and post entry events for pois that aren't already in our list
+    // also create the new list of regions that the user is in
+    Set<String> updatedRegionsUserIsIn = new HashSet<String>();
+    for (PlacesPOI poi : nearbyPois) {
+        // check if the user is in this poi
+        if (poi.containsUser()) {
+            // the user is in the poi, now we need to make sure we haven't already recorded this entry event
+            if (!regionsUserIsAlreadyIn.contains(poi.getIdentifier())) {
+                Geofence poiGeofence = new Geofence.Builder()
+                    .setRequestId(poi.getIdentifier())
+                    .setTransitionTypes(Geofence.GEOFENCE_TRANSITION_ENTER)
+                    .setCircularRegion(poi.getLatitude(), poi.getLongitude(), poi.getRadius())
+                    .setExpirationDuration(Geofence.NEVER_EXPIRE)
+                    .build();
+                Places.processGeofence(poiGeofence, Geofence.GEOFENCE_TRANSITION_ENTER);
+            }
+
+            // add the region to our new list of regions
+            updatedRegionsUserIsIn.add(poi.getIdentifier());
+        }
+    }
+
+    // update SharedPreferences with our new list of regions
+    SharedPreferences.Editor editor = getApplicationContext().getSharedPreferences("places", 0).edit();
+    editor.putStringSet("regionsUserIsAlreadyIn", updatedRegionsUserIsIn).apply();
+}
+```
+{% endtab %}
+
+{% tab title="iOS (Objective-C)" %}
+
+This code sample shows handling the result provided in the callback of `getNearbyPointsOfInterest:limit:callback:errorCallback:`, an `NSArray<ACPPlacesPoi *> *`:
```

Comment:
Please reword to:
"The following code sample shows the handling of the result that was provided..."